Packs/SumoLogic_Cloud_SIEM/Integrations/SumoLogicCloudSIEM/SumoLogicCloudSIEM.yml

```diff
@@ -182,8 +216,32 @@ script:
     - contextPath: SumoLogicSec.InsightComments.InsightId
       description: The ID of the Insight.
       type: String
-    description: Get comments for a specific Insight ID (Users can post and update
-      comments on the Sumo Logic Cloud SIEM portal for any Insight ID).
+    description: Get comments for a specific Insight ID (Users can post and update comments on the Sumo Logic Cloud SIEM portal for any Insight ID).
+  - name: sumologic-sec-insight-add-comment
```

Comment:
Please add the command to the readme integration file

Reply:
added